susemanager-utils/testing/automation/update-obs-environment-cr-project.sh

```diff
@@ -1,15 +1,93 @@
 #!/bin/bash
 
-if [ $# -ne 1 ];then
-    echo "usage: $0 N"
+parent_project="systemsmanagement:Uyuni:Master"
+
+usage_and_exit() {
+    echo "usage: $0 N [test_project]"
```

Comment:
```suggestion
    echo "usage: ${0} N [test_project]"
```
Same for all other variables missing `{}`